src/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -81,7 +81,7 @@ internal static AuthorityInfo FromAuthorityUri(string authorityUri, bool validat
         internal static AuthorityInfo FromAadAuthority(Uri cloudInstanceUri, Guid tenantId, bool validateAuthority)
         {
 #pragma warning disable CA1305 // Specify IFormatProvider
-            return FromAuthorityUri(string.Format(CultureInfo.InvariantCulture, "{0}/{1}/", cloudInstanceUri, tenantId.ToString("D")), validateAuthority);
+            return FromAuthorityUri(string.Format(CultureInfo.InvariantCulture, "{0}{1}/", cloudInstanceUri, tenantId.ToString("D")), validateAuthority);
```

Comment:
While your solution works, it's still a bit cryptic because developers do not know how Uri behaves. Consider using the Uri ctor instead: 

```csharp
new Uri(Uri baseUri, string relativeUri)
```

see https://stackoverflow.com/questions/372865/path-combine-for-urls

Reply:
This is much more elegant. Thanks!